h2o-r/tests/testdir_algos/glm/runit_pubdev_8683_glm_gamma_dispersion_factor_large.R

```diff
@@ -28,8 +28,8 @@ test_glm_gammas <- function() {
 
 generate_dataset<-function(f1R, numRows, numCols, pow, phi, mu) {
   resp <- tweedie::rtweedie(numRows, xi=pow, mu, phi, power=pow)
-  f1h2o <- as.h2o.data.frame(f1R)
-  resph2o <- as.h2o.data.frame(as.data.frame(resp))
+  f1h2o <- as.h2o(f1R)
```

Comment:
`as.h2o` is a generic that we expose in the R package. `as.h2o.data.frame` is an implementation for `data.frame` that is not directly exposed by the package. Recent changes in h2o R package force the tests to run as if the code would be run by the user so we can find out if we forgot to expose some function/method.